Dockerfile

```diff
@@ -1,24 +1,43 @@
-FROM cgal/testsuite-docker:archlinux
+FROM cgal/testsuite-docker:ubuntu
 MAINTAINER Philipp Moeller <bootsarehax@gmail.com>
+RUN  apt-get update \
+  && apt-get install -y bison \
+     git \
+     flex \
+     graphviz \
+     python3 \
+     python3-pyquery \
+     texlive-binaries \
+  && apt-get clean -y
 
-RUN pacman -Syy && pacman -S --noconfirm \
-    bison \
-    flex \
-    git \
-    graphviz \
-    python2 \
-    python2-pyquery \
-    texlive-bin \
-    && \
-    pacman -Scc --noconfirm
+RUN mkdir /doxygen
+WORKDIR /doxygen
+
+RUN git clone https://github.com/CGAL/doxygen.git cgal_dox && \
+    mkdir cgal_1_8_13 && \
+    cd cgal_dox && \
+    git checkout release_1_8_13_patched && \
+    mkdir build && \
+    cd build && \
+    cmake .. && \
+    make && \
+    cp bin/doxygen ../../cgal_1_8_13 && \
+    cd ../../ && rm -rf cgal_dox
+
+
+
+RUN git clone https://github.com/doxygen/doxygen.git off_dox && \
+    mkdir cgal_1_9_1 && \
+    cd off_dox && \
+    git checkout Release_1_9_1 && \
```

Comment:
That is not correct. We want a version of Doxygen 1.9.1 that is modified by the same patch as our custom version of Doxygen.

@sloriot Would you mind preparing a branch or a tag `release_1_9_1_patched` in https://github.com/CGAL/doxygen.git?

Reply:
Done. Did not pick the xml fix as it seems no longer needed.